frontend/src/app/modules/user/profile-page-main-info/profile-page-main-info.component.ts

```diff
@@ -18,6 +27,10 @@ export class ProfilePageMainInfoComponent implements OnChanges {
 
     @Input() isFriend: Boolean;
 
+    @Input() currentUser: IUser;
+
+    @Input() friendshipId: number | undefined;
```

Comment:
Maybe 'friendshipId?' instead of '| undefined'?

Reply:
we receive data from in that format, so I defined it as a return type

Reply:
With a question mark the variable will be able to hold a normal value, null and undefined